owncloudApp/src/main/res/values/strings.xml

```diff
@@ -726,20 +726,6 @@
     <string name="release_notes_proceed">Proceed</string>
     <string name="release_notes_icon">Release note icon</string>
 
-    <string name="release_notes_4_3_0_title_accessibility_improvements">Improvements in accessibility</string>
-    <string name="release_notes_4_3_0_subtitle_accessibility_improvements">Some improvements to make the application more accessible</string>
-    <string name="release_notes_4_4_0_title_improved_from_original_folder_auto_upload">Auto uploads more consistent</string>
-    <string name="release_notes_4_4_0_subtitle_improved_from_original_folder_auto_upload">Added some mechanisms over retries to improve reliability</string>
-    <string name="release_notes_4_4_0_title_resharing_capability">Improvements in private shares</string>
-    <string name="release_notes_4_4_0_subtitle_resharing_capability">The \"can share\" option is now only shown when editing a private share if the user is able to edit it</string>
-    <string name="release_notes_bugfixes_title">Minor bugfixes</string>
-    <string name="release_notes_bugfixes_subtitle">Some bugs were fixed to improve experience in the app</string>
```

Comment:
Let's not remove these 2 strings since they are generic and useful for every release 😃

Reply:
Added 👍